src/Hosting/Hosting/test/HostBuilderTests.cs

```diff
@@ -118,10 +118,8 @@ public void CanConfigureAppConfigurationAndRetrieveFromDI()
         [Fact]
         public void CanConfigureAppConfigurationFromFile()
         {
-            // Needs to look in the project directory like VS would
-            var projectDir = Path.Combine(TestPathUtilities.GetRepoRootDirectory(), "src", "Hosting", "Hosting", "test");
             var hostBuilder = new HostBuilder()
-                .UseContentRoot(projectDir)
+                .UseContentRoot(AppContext.BaseDirectory)
```

Comment:
Isn't `AppContext.BaseDirectory` normally the bin/Debug/{framework} directory?

Reply:
It's the directory that contains the test assembly.

Reply:
Then this update will change the content root significantly. That may be your intent (given the update to copy everything to build and publish directories) but could break things looking for non-`@(Content)` files.

Reply:
That is my intent. Those tests should update to use `<Content>`. See the description in https://github.com/aspnet/Extensions/issues/1697 about what we need to do to tests like this.